packages/playwright-core/src/utils/stackTrace.ts

```diff
@@ -87,12 +88,8 @@ export function captureStackTrace(rawStack?: string): ParsedStackTrace {
       return null;
     if (isInternalFileName(frame.file, frame.function))
       return null;
-    // Workaround for https://github.com/tapjs/stack-utils/issues/60
-    let fileName: string;
-    if (frame.file.startsWith('file://'))
-      fileName = new URL(frame.file).pathname;
```

Comment:
lets see what the bots say, on my mac it passed. it was fixed in 2019, so its fine, since we only support Node.js 14 officially which was released in 2020.